### DISCUSSION TITLE
How do you remove seconds from DateTime?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#3258] [Q&A] [answered]
I have no need to display the seconds for [DateTime ](https://preview.blazorise.com/docs/components/date). I need the minutes but would love to be able to hide the seconds. Isn't that possible somehow?

<img width="409" alt="image" src="https://user-images.githubusercontent.com/2386572/147420416-81c83d62-f4f2-46c0-8981-f393820009b5.png">

**Comment by David-Moreira** [comment, marked as answer]
`DateEdit` uses native html input for date, which means it can have some limitations.
That means a custom implementation is required for more control. We do have a date component that implements https://flatpickr.js.org/
which has more features.


This might be what you're looking for, let us know if it helps!
`<DatePicker TValue="DateTime" InputMode="DateInputMode.DateTime" TimeAs24hr></DatePicker>`

https://preview.blazorise.com/docs/components/date-picker

**Reply by sturlath** [reply]
Ok thanks, it looks like I just need to have two separate controls then. One with the date and one with hours/min picker.

But why would you ever need seconds in a date select? humm... no way to loose it?

**Reply by stsrki** [reply]
@sturlath There is no 100% proof code to lose the seconds. It all depends on the browser and system setting and we cannot control that.

**Reply by sturlath** [reply]
I was going to use two controls, [DateEdit ](https://preview.blazorise.com/docs/components/date) and a [Time ](https://preview.blazorise.com/docs/components/time) but I see that Time also has seconds so that approach will not work.

But I see that  https://flatpickr.js.org  has a [TimePicker ](https://flatpickr.js.org/examples/#time-picker) that doesn't display seconds.
Any change of getting that into Blazor?

![image](https://user-images.githubusercontent.com/2386572/147470631-cc537c29-b974-4081-a731-c1ec416886e4.png)

I can live with the seconds for now but I would love to loose them at some point.

**Reply by stsrki** [reply]
We already have TimePicker based on flatpickr: https://blazorise.com/docs/components/time-picker

**Reply by sturlath** [reply]
Ah sorry.. working to much.. great stuff! Will use that ;-)

**Reply by David-Moreira** [reply]
Are you sure you need two different components? That's indeed a solution but I think I tested the one I linked you, and it did not have seconds... 
`<DatePicker TValue="DateTime" InputMode="DateInputMode.DateTime" TimeAs24hr></DatePicker>`
Did you actually try this one? Or it is not what you are looking for?
